import-service/serverless.ts

```diff
@@ -26,6 +26,7 @@ const serverlessConfiguration: Serverless = {
     },
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      SQS_URL: 'https://sqs.eu-west-1.amazonaws.com/314054664759/catalog-items-queue'
```

Comment:
Consider using sharing features of CloudFormation/Serverless instead of hardcoded values